webui/react/src/components/kit/utils/color.ts

```diff
@@ -1,4 +1,4 @@
-import { GLASBEY } from 'constants/colors';
+import { GLASBEY } from 'components/kit/constants/colors';
```

Comment:
not sure if it makes sense to import/export all of these color utils from the UI kit.

It makes sense to do that for things that are specific to the UI kit, like the theme variables.

But utils for handling glasbey or hsl/rgba/hex, etc. are not really tied to the UI kit imho, and would make more sense being both utils used by the App and, for the UI kit, an "internal" util.